src/containers/team-leaderboard/ducks/thunks.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  LeaderboardItem,
+  LeaderboardThunkAction,
+} from '../../../components/leaderboard/ducks/types';
+import { 
+  getLeaderboardAction
+} from '../../../components/leaderboard/ducks/actions';
+
+export const getTeamsLeaderboard = (
+  previousDays: number,
+): LeaderboardThunkAction<void> => {
+  const action = getLeaderboardAction(previousDays);
```

Comment:
We shouldn't have to have separate actions for fetching different ranges of data, we should have one action which loads the range of data as requested by the user.

Reply:
When I had one action before, it was not making a distinction between each of the ranges, so everything got set to the all time data since that was the once dispatched last. However this won't be an issue if we change this to only store one tab's data at a time